<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-04T14:30:43 IST" name="MySuite" finished-at="2023-05-04T14:30:51 IST" duration-ms="8201">
    <groups>
    </groups>
    <test started-at="2023-05-04T14:30:43 IST" name="Regression" finished-at="2023-05-04T14:30:51 IST" duration-ms="8201">
      <class name="pom.tc.LoginTest">
        <test-method is-config="true" signature="setup(org.testng.xml.XmlTest)[pri:0, instance:pom.tc.LoginTest@58ebfd03]" started-at="2023-05-04T14:30:43 IST" name="setup" finished-at="2023-05-04T14:30:48 IST" duration-ms="4928" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.xml.XmlTest@78f0ec71]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="LoginTestMethod()[pri:0, instance:pom.tc.LoginTest@58ebfd03]" started-at="2023-05-04T14:30:48 IST" name="LoginTestMethod" finished-at="2023-05-04T14:30:51 IST" duration-ms="2855" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTestMethod -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:pom.tc.LoginTest@58ebfd03]" started-at="2023-05-04T14:30:51 IST" name="tearDown" finished-at="2023-05-04T14:30:51 IST" duration-ms="178" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- pom.tc.LoginTest -->
      <class name="pom.tc.LoginGetItemsTest">
        <test-method is-config="true" signature="setup(org.testng.xml.XmlTest)[pri:0, instance:pom.tc.LoginGetItemsTest@5b07730f]" started-at="2023-05-04T14:30:43 IST" name="setup" finished-at="2023-05-04T14:30:48 IST" duration-ms="5002" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.xml.XmlTest@78f0ec71]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="LoginGetItemsTestMethod()[pri:0, instance:pom.tc.LoginGetItemsTest@5b07730f]" started-at="2023-05-04T14:30:48 IST" name="LoginGetItemsTestMethod" finished-at="2023-05-04T14:30:51 IST" duration-ms="2875" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [5] but found [6]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [5] but found [6]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at pom.tc.LoginGetItemsTest.LoginGetItemsTestMethod(LoginGetItemsTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginGetItemsTestMethod -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:pom.tc.LoginGetItemsTest@5b07730f]" started-at="2023-05-04T14:30:51 IST" name="tearDown" finished-at="2023-05-04T14:30:51 IST" duration-ms="123" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- pom.tc.LoginGetItemsTest -->
    </test> <!-- Regression -->
  </suite> <!-- MySuite -->
</testng-results>
